wandb/sdk/launch/sweeps/utils.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import wandb
+from wandb import util
+from wandb.sdk.launch.utils import LaunchError
+
+
+def load_launch_sweep_cli_params(
+    launch_config: Optional[str], queue: Union[str, None]
+) -> Tuple[Dict[str, Any], str]:
+    if launch_config:
+        parsed_config = util.load_json_yaml_dict(launch_config)
+        if parsed_config is None:
+            raise LaunchError(f"Invalid format for launch config at {launch_config}")
+        wandb.termlog(f"Using launch 🚀 with config: {parsed_config}")
+    else:
+        parsed_config = {}
+
+    queue = queue or parsed_config.get("queue")
+
+    return parsed_config, queue
+
+
+def construct_scheduler_entrypoint(
+    sweep_config: Dict[str, Any],
+    queue: str,
+    project: str,
+    num_workers: int,
+) -> Optional[List[str]]:
+    """Construct a sweep scheduler run spec.
+
+    logs error and returns None if misconfigured, otherwise returns entrypoint
+    """
+    job = sweep_config.get("job")
+    image_uri = sweep_config.get("image_uri")
+    if not job and not image_uri:  # don't allow empty string
+        wandb.termerror("No 'job' nor 'image_uri' found in sweep config")
+        return []
+    elif job and image_uri:
+        wandb.termerror("Sweep has both 'job' and 'image_uri'")
```

Comment:
could you give more informative information, what does it mean? are not scheduling this sweep? should the user provide only one of these?